robotframework_reportportal/variables.py

```diff
@@ -62,6 +62,8 @@ def __init__(self):
             'RP_SKIPPED_ISSUE', default='True'))
         self.test_attributes = get_variable(
             'RP_TEST_ATTRIBUTES', default='').split()
+        self.verify_ssl = strtobool(get_variable(
```

Comment:
I would also like to see a unit-test for the new property. Check out 'skipped_not_issue' PR as example: https://github.com/reportportal/agent-Python-RobotFramework/pull/95